src/image/mod.rs

```diff
@@ -227,6 +227,12 @@ impl<'de> Deserialize<'de> for Os {
     }
 }
 
+impl Default for Os {
+    fn default() -> Self {
+        Os::from(std::env::consts::OS)
```

Comment:
I spot checked this and it seems like there's enough overlap with what Rust has in https://doc.rust-lang.org/std/env/consts/constant.OS.html and what OCI (really Go I guess) expects.

OK I just got a link to https://github.com/containerd/containerd/blob/140ecc9247386d3be21616fe285021c081f4ea08/platforms/database.go#L83
which seems to be a key player in this.